index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My portfolio</title>
+    <link href="styles.css" rel="stylesheet">
+    <link rel="preconnect" href="https://fonts.googleapis.com"><link rel="preconnect" href="https://fonts.gstatic.com" crossorigin><link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet">
+    <script src="https://kit.fontawesome.com/dcdd89c74c.js" crossorigin="anonymous"></script>
+</head>
+<body>
+    <div class="hdiv">
+        <header class ="head">Uthman</header>
+        <i class="fa-solid fa-bars menu-bar"></i> 
+    </div> 
+    <section class="section1">
+    <h1 class="h">I'm Uthman <br> Glad to see <br>you!</h1> 
+    <p class="info1">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+    <p class="lc info2">LET’S CONNECT</p>
+    <footer class="foot">
+        <div class="left">
+            <ul class="footer-items">
+            <li class="dotless twitter"><img src='images/Vector.svg' alt="twitter icon"></li>
+            <li class="dotless insta"><img class="iinsta" src='images/i.png'><img class="ninsta" src='images/n.png' alt="instagram icon"></li>
+            <li class="dotless m"><img src='images/M.svg' alt="m icon"></li>
+            <li class="dotless gh"><img src='images/gitHub.svg' alt="github icon"></li>
+            <li class="dotless peace"><img src='images/peace.svg' alt="peace icon"></li>
+            </ul>
+        </div>
+    </footer>
```

Comment:
- [ Required ] Please consider removing the `footer` tag from this section since this is not the footer part of this web page. In future project milestones, you'll add other sections to this web page so please remove the `footer` tag from this part of the webpage for now.